checkov/terraform/checks/data/gcp/GooglePolicyIsPrivate.py

```diff
@@ -0,0 +1,37 @@
+from typing import Dict, List, Any
+
+from checkov.terraform.checks.data.base_check import BaseDataCheck
+from checkov.common.models.enums import CheckResult, CheckCategories
+
+
+class GooglePolicyIsPrivate(BaseDataCheck):
+    def __init__(self) -> None:
+        """
+        ensure policy defined is not public
+        policy should not set 'allUsers' or 'allAuthenticatedUsers' in the attribute 'member'/'members'
+        """
+        name = "Policy should not define public access"
+        id = "CKV_GCP_113"
+        supported_data = ["google_iam_policy"]
+        categories = [CheckCategories.IAM]
+        super().__init__(name=name, id=id, categories=categories, supported_data=supported_data)
+
+    def scan_data_conf(self, conf: Dict[str, List[Any]]) -> CheckResult:
+        """
+            validates gcp policy
+        """
+        fails = ["allUsers", "allAuthenticatedUsers"]
+        if conf.get("binding") and isinstance(conf.get("binding"), list):
+            bindings = conf.get("binding")
+            for binding in bindings:
+                if binding.get("members") and isinstance(binding.get("members"), list):
```

Comment:
Shouldn't there be a check for `member` argument as well?

Reply:
No I don't think so, there is no member argument in that data source: https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/iam_policy